Package:     fox1.6
Subject:     setting compile flags
Author:      Joachim Wiedorn <ad_debian at joonet.de>
Forwarded:   no
Last-Update: 2012-05-17

[Hardening] For the use of dpkg-buildflags all compile flags made by
Debian must be added to configure.ac before creating all Makefiles.
---

diff -urN s06/configure.ac s07/configure.ac
--- s06/configure.ac	2012-05-13 10:28:46.000000000 +0200
+++ s07/configure.ac	2012-05-17 16:06:58.099533663 +0200
@@ -56,8 +56,9 @@
 AC_SUBST(LT_REVISION)
 AC_SUBST(LT_AGE)
 
-# Prevents setting flags.
-CXXFLAGS=""
+# Setting flags of Debian rules
+CXXFLAGS=${CXXFLAGS}
+LDFLAGS=${LDFLAGS}
 
 # More secret source
 AC_GNU_SOURCE
